### PR TITLE
Added calculator and portfolio projects in folder 'Kritika'

### PR DESCRIPTION
**Pull request by KritikaSachdeva**
Please review both the added projects.

### CODE REVIEW

File: Kritika_Sachdeva/profile/index.html [hunk 1]
```diff
@@ -14,14 +14,16 @@
...
+
 </head>
 <body>
 
 <!--HEADER-->
 <section id="header">
     <div class="container text-center">
         <div class="user-box">
-            <img src="IMG-4740.JPG">
+            <img src="pp.jpg">
```
Comment (nikhilrstg18) [inline Kritika_Sachdeva/profile/index.html]: 1. folder structure missing 
all static assets must be in a folder named `assets`
For Eg. 

/project
---/assets
------/css
------/scipts
---index.html

File: Kritika_Sachdeva/profile/index.html [hunk 2]
```diff
@@ -69,7 +71,7 @@ <h1>KRITIKA SACHDEVA</h1>
       <div class="about-me container" id="about-me">
       <div class="row">
           <div class="col-md-6 text-center">
-              <img src="IMG-4740.jpg" class="profile-img">
+              <img src="IMG-4469.jpg" class="profile-img">
```
Comment (nikhilrstg18) [inline Kritika_Sachdeva/profile/index.html]: same as above

File: Kritika_Sachdeva/profile/mystyles.css [hunk 3]
```diff
@@ -57,7 +57,7 @@ body{
 }
 
 .profile-img{
-    height: 400px;
+    height: 420px;
```
Comment (nikhilrstg18) [inline Kritika_Sachdeva/profile/mystyles.css]: is this responsive?

File: Kritika_Sachdeva/profile/index.html [hunk 4]
```diff
@@ -40,7 +42,7 @@ <h1>KRITIKA SACHDEVA</h1>
     <nav class="navbar navbar-expand-lg">
         <a class="navbar-logo" href="#"><img src="IMG-4740.jpg"></a>
```
Comment (nikhilrstg18) [inline Kritika_Sachdeva/profile/index.html]: same as above

File: Kritika_Sachdeva/Calculator/styles.css [hunk 5]
```diff
@@ -0,0 +1,122 @@
+body{
+    background-image: linear-gradient(to right, black, lightblue);
+    margin: 0;
+
+}
+
+.container{
+    background-color: rgba(0, 0, 0, 0.842);
+    height: 450px;
```
Comment (nikhilrstg18) [inline Kritika_Sachdeva/Calculator/styles.css]: is this responsive design?

File: Kritika_Sachdeva/Calculator/styles.css [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+    width: 266px;
+    margin: 2rem;
+    top: 25px;
+    border-radius: 4px;
+    
+}
+
+.scrn input{
+    position: relative;
+    left: 2px;
```
Comment (nikhilrstg18) [inline Kritika_Sachdeva/Calculator/styles.css]: for responsive designs start using relative units, px won't help you with responsiveness

File: Kritika_Sachdeva/Calculator/index.html [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+</head>
+<body>
+    
+    <div class="container">
+        <div class="scrn"><form name="calc"><input type="text" name="display" id="scn" size="20" readonly></form></div>
+
+        <div class="calc_btns">
+            <p>
+                <input type="button" value="x!" class="other_opr" onclick=fact();>
+                <input type="button" value="C" class="other_opr" onclick="calc.display.value=''">
```
Comment (nikhilrstg18) [inline Kritika_Sachdeva/Calculator/index.html]: always remember to self evaluate your code with SOLID DRY KISS 

You are writing JS directly in HTML, violating the Single Responsibility Principle here.
Please move all JS code into a separate .js file in /assets/js/

File: Kritika_Sachdeva/profile/index.html [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+    <div class="container text-center">
+        <div class="user-box">
+            <img src="pp.jpg">
+            <h1>KRITIKA SACHDEVA</h1>
+            <p>Learner. Believer. Complex.</p>
+        </div>
+    </div>
+
+    <div class="scroll-btn">
+        <div class="scroll-bar"><a href=""> <span></span></a></div>
```
Comment (nikhilrstg18) [inline Kritika_Sachdeva/profile/index.html]: what's the purpose of empty link here?

File: Kritika_Sachdeva/profile/index.html [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+            <li class="nav-item">
+                <a class="nav-link " href="#">CONTACT</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </div>
+
+
+      <!--About-me-->
```
Comment (nikhilrstg18) [inline Kritika_Sachdeva/profile/index.html:63]: Good to see that for each section you have added comments in HTML layout

File: Kritika_Sachdeva/profile/mystyles.css [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+    padding-bottom: 50px;
+}
+
+.profile-img{
+    height: 420px;
+    width: 350px;
+}
+
+.nav-bar{
+    position: sticky;
```
Comment (nikhilrstg18) [inline Kritika_Sachdeva/profile/mystyles.css]: this will fail sticky behavior in IE9 and IE11

**Review by nikhilrstg18 — CHANGES_REQUESTED** [review]
Missing SOLID DRY KISS in code
Missing responsiveness in CSS

**Review by nikhilrstg18 — CHANGES_REQUESTED** [review]
@KritikaSachdeva 
reply on each PR review comment 
there were some deviations in your calculator app, please fix them, test it then provide a demo

File: Kritika_Sachdeva/profile/index.html (hunk 1 above)
Comment (KritikaSachdeva) [reply Kritika_Sachdeva/profile/index.html]: Done!

File: Kritika_Sachdeva/Calculator/styles.css (hunk 6 above)
Comment (KritikaSachdeva) [reply Kritika_Sachdeva/Calculator/styles.css]: Done!

File: Kritika_Sachdeva/Calculator/styles.css (hunk 5 above)
Comment (KritikaSachdeva) [reply Kritika_Sachdeva/Calculator/styles.css]: No, it wasn't.
I've made the required changes.

File: Kritika_Sachdeva/profile/index.html (hunk 1 above)
Comment (KritikaSachdeva) [reply Kritika_Sachdeva/profile/index.html]: Made the required changes.

File: Kritika_Sachdeva/profile/index.html (hunk 2 above)
Comment (KritikaSachdeva) [reply Kritika_Sachdeva/profile/index.html]: Done.

File: Kritika_Sachdeva/profile/index.html (hunk 4 above)
Comment (KritikaSachdeva) [reply Kritika_Sachdeva/profile/index.html]: Done

File: Kritika_Sachdeva/Calculator/index.html (hunk 7 above)
Comment (KritikaSachdeva) [reply Kritika_Sachdeva/Calculator/index.html]: Okay. I ll surely update it .

File: Kritika_Sachdeva/profile/index.html (hunk 9 above)
Comment (KritikaSachdeva) [reply Kritika_Sachdeva/profile/index.html:63]: Thankyou.

File: Kritika_Sachdeva/profile/mystyles.css (hunk 3 above)
Comment (KritikaSachdeva) [reply Kritika_Sachdeva/profile/mystyles.css]: Modified .

File: Kritika_Sachdeva/profile/index.html (hunk 8 above)
Comment (KritikaSachdeva) [reply Kritika_Sachdeva/profile/index.html]: It was supposed to be linked to other section on the same page.
Modified the code accordingly.

**Review by nikhilrstg18 — APPROVED** [review]
looks good to me,
replied and worked on all PR review comments

**Review by amandeeprikhi — APPROVED** [review]
LGTM

**Pull request merged**